src/modules/modCommands/commands/autoresponse.js

```diff
@@ -72,17 +72,17 @@ async function fun(client, message, args) {
                     const old = acc[acc.length - 1];
                     let newEmbed = old;
                     if (
-                        old.fields.length >= 25 ||
+                        old.data.fields?.length >= 25 ||
                         countEmbedCharacters(old) + value.length + name.length > 6000
                     ) {
-                        newEmbed = new MessageEmbed();
+                        newEmbed = new EmbedBuilder();
                         acc.push(newEmbed);
                     }
-                    newEmbed.addField(name, value, true);
+                    newEmbed.addFields({ name, value, inline: true });
```

Comment:
this line was the one which threw the "invalid number something" error at me [testing needed]

Reply:
Something is broken in not my code. If you can replicate it, then I'll consider unresolving this.